### ISSUE TITLE
 brews with custom brew method can't be saved

### ISSUE THREAD

**Issue by abaebae** [graphefruit/Beanconqueror#641]
I created a custom brew method (for Pulsar). Everything worked fine. But since the last update I can no longer save brews with the custom brew method. After clicking save they just vanish. Selecting another brew method first (e.g. V60) and saving works. Editing the brew afterwards to change the method to my custom one does not let me save it (changing to Aeropress and saving works).
Also seems to have deleted some of my previous brews with my custom pulsar method. I don't see any errors in the logs. Tried it with different beans. If I can provide more informations let me know

```json
[
  {
    "key": "Log",
    "timestamp": "21.12.2023 10:10:01:081",
    "value": "Check app ready - Already loaded, no interval needed"
  },
  {
    "key": "Log",
    "timestamp": "21.12.2023 10:10:01:103",
    "value": "SUCCESS - Track Page - /home/dashboard"
  },
  {
    "key": "Log",
    "timestamp": "21.12.2023 10:10:03:405",
    "value": "SUCCESS - Track event page - Category:BREW Action:ADD"
  },
  {
    "key": "Log",
    "timestamp": "21.12.2023 10:10:07:894",
    "value": "SUCCESS - Track event page - Category:BREW Action:ADD_FINISH"
  },
  {
    "key": "Log",
    "timestamp": "21.12.2023 10:10:07:913",
    "value": "SUCCESS - Track Page - /home/brews"
  },
  {
    "key": "Log",
    "timestamp": "21.12.2023 10:10:09:715",
    "value": "SUCCESS - Track Page - /home/dashboard"
  },
  {
    "key": "Log",
    "timestamp": "21.12.2023 10:10:12:115",
    "value": "Check app ready - Already loaded, no interval needed"
  },
  {
    "key": "Log",
    "timestamp": "21.12.2023 10:10:12:138",
    "value": "SUCCESS - Track Page - /info"
  },
  {
    "key": "Log",
    "timestamp": "21.12.2023 10:10:13:313",
    "value": "SUCCESS - Track Page - /info/logs"
  }
]
```

**Comment by graphefruit** [comment]
Hey @abaebae,
thanks for the report and sorry to hear this.
Could you please goto "Menu" -> Export and send me the .zip file to: info@beanconqueror.com?

Also please provide me the information:
Android / iOS
Which OS Version is installed
Custom Rom?

I'll have a look this evening.
Thanks
Lars

**Comment by graphefruit** [comment]
Hey there.
Thanks for the ticket.
I've released Version 7.0.1 with the fix for it on iOS aswell as android.
The issue was the deactivated timer-parameter.
Version 7.0 had a feature which stopped timer on saving, or on toggling the preparations which caused this issue when a timer was not existing.
I close this ticket now.
Best & Thanks for reporting
Lars
